src/app/courses/page.tsx

```diff
@@ -1,11 +1,101 @@
+'use client'
+
+import styled from '@emotion/styled'
 import { COURSES } from '@/data/courses'
+import { FontVariant, Color } from '@/app/theme'
 
 export default function Page() {
   return (
     <main>
-      {COURSES.map(({ title }, i) => (
-        <div key={i}>{title}</div>
-      ))}
+      <h1>Courses</h1>
+      <Sections>
+        {COURSES.map(({ code, title, description, editions }, i) => (
+          <>
+            <Section key={i}>
+              <SectionTitle>
+                {code} {title}
+              </SectionTitle>
+              <SectionContent>{description}</SectionContent>
+              <SectionGroup>
+                {groupEditions(editions).map((editionGroup, i) => (
+                  <SectionEditions key={i}>
+                    {editionGroup.map((edition: any, i: any) => (
+                      <SectionEdition key={i} href={edition.url}>
+                        {' '}
```

Comment:
was `{' '}` intended? if you need spacing, I suggest using margin or padding

Reply:
Oh i didn't intend it, I think 'npx prettier' did it automatically.. I fixed it!